src/js/constants.js

```diff
@@ -101,6 +101,8 @@ export const URLS = {
   USERS: "get-users-list",
 };
 
+export const miningLayers = ["cMines", "nMines", "pMines"];
+
```

Comment:
Would be nice to define it in terms of available layers constant (the first 3 in this case)